app/views/fields/checkbox-options.test.js

```diff
@@ -163,8 +167,9 @@ describe('checkboxOptions', () => {
       .then((res) => {
         const $ = cheerio.load(res.text);
 
-        expect($('.nhsuk-error-message').text().trim()).toEqual('Error: Some error message');
-        expect($('#fieldId-error').length).toEqual(1);
+        const question = $('div[data-test-id="question-fieldId"]');
+        expect(question.find('.nhsuk-error-message').text().trim()).toEqual('Error: Some error message');
+        expect(question.find('div[data-test-id="checkbox-options-error"]').length).toEqual(1);
```

Comment:
Just a minor preference but I would prefer the check it exists before the check of the error message. Upto you if you want to change it

Reply:
Yep good shout. I prefer that.